lldb/source/Plugins/Language/CPlusPlus/LibStdcpp.cpp

```diff
@@ -254,13 +254,13 @@ bool lldb_private::formatters::LibStdcppStringSummaryProvider(
   } else
     addr_of_string =
         valobj.GetAddressOf(scalar_is_load_addr, &addr_type);
-  if (addr_of_string != LLDB_INVALID_ADDRESS) {
+  if (addr_of_string != LLDB_INVALID_ADDRESS ||
+        addr_type == eAddressTypeHost) {
```

Comment:
Do we need this since we iterate all cases anyway?

Reply:
Yes, the switch statement is within the if block, if we don't add this exception we will return false and fall back to the ValueObjectPrinter

Reply:
Ohh wow, so for this specific situation we will have `addr_of_string == LLDB_INVALID_ADDRESS`?
Would you mind adding a comment for that?